package.json

```
@@ -2,38 +2,52 @@
   "name": "web-streams-polyfill",
   "version": "3.1.0",
   "description": "Web Streams, based on the WHATWG spec reference implementation",
-  "main": "dist/polyfill",
-  "browser": "dist/polyfill.min.js",
+  "main": "dist/polyfill.js",
+  "browser": "dist/polyfill.js",
   "module": "dist/polyfill.mjs",
   "types": "dist/types/polyfill.d.ts",
-  "typesVersions": {
-    ">=3.6": {
-      "dist/types/*": [
-        "dist/types/ts3.6/*"
-      ]
-    }
+  "exports": {
+    ".": {
+      "import": "./dist/polyfill.mjs",
+      "require": "./dist/polyfill.js"
+    },
+    "./es6": {
+      "import": "./dist/polyfill.es6.mjs",
+      "require": "./dist/polyfill.es6.js"
+    },
```

>     1.6kB  dist/polyfill.es6.mjs
>     1.6kB  dist/polyfill.mjs

These files are small, but useless. When imported, UMD build should have no issue setting globals.

```suggestion
    ".": "./dist/polyfill.js"
    "./es6": "./dist/polyfill.es6.js",
```

---

Right now, the polyfill *also* re-exports everything from the ponyfill, so you *could* use `import { ReadableStream } from 'web-streams-polyfill'` (without the `/ponyfill`) if you wanted to.

Maybe I should remove that, and *only* allow `import from 'web-streams-polyfill'` (i.e. side effect only). Not sure. 🤔

---

Okay, I see. If we follow [your other suggestion](https://github.com/MattiasBuelens/web-streams-polyfill/pull/83#discussion_r690372483) of making the *ponyfill* the default, then it's much less likely that a user would write:
```javascript
import { ReadableStream } from 'web-streams-polyfill/polyfill';
```
Then it makes sense to have *no* exports on the `/polyfill` variant. It would *only* mutate `globalThis`, and so would be valid as both a regular script and as an ES module.